docker/ha/orion.conf

```diff
@@ -7,7 +7,7 @@ clientnetworkinterface = "0.0.0.0"
 publickeys = ["/var/keys/nodeKey.pub"]
 privatekeys = ["/var/keys/nodeKey.key"]
 passwords = "/var/keys/passwordFile"
-workdir = "/data"
+workdir = "/tmp"
```

Comment:
Can make a custom image and use `/data` after its been created by root and permissions changed, or just use `/tmp` here like this? Im not in favour of the former because it doesnt provide value, users would mount a volume anyway